.github/workflows/test.yaml

```diff
@@ -82,10 +82,9 @@ jobs:
         run: cmake -S . -B build
       - name: Build, debug configuration
         run: cmake --build build --config Debug
-      - name: Client-server test, debug
-        run: python tests/client-server.py build/tests/Debug/client.exe build/tests/Debug/server.exe
-      - name: Check static library list
-        run: python tests/verify-static-libraries.py
+        env:
+          CLIENT_BINARY: build/tests/Debug/client.exe
+          SERVER_BINARY: build/tests/Debug/server.exe
```

Comment:
The updated `client_server.rs` takes these inputs as env vars instead of CLI args because there isn't a way to pass CLI args to a Rust integration test unless we disable the built-in runner and implement our own `main`. It didn't seem worth it to me in this case.